### PR TITLE
Manual blocks

### PR DESCRIPTION
**Pull request by Takhion**

### CODE REVIEW

File: android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java [hunk 1]
```diff
@@ -223,23 +245,98 @@ private static void transformJson(
                         dateParser, dateFormatterSessions);
 
         sessions.add(session);
+    }
+
+    private static void addBlocksManually(@NonNull Collection<Block> blocks)
```
Comment (rock3r) [inline android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java:250]: Bruteforce ftw :dancer:

File: android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java [hunk 2]
```diff
@@ -223,23 +245,98 @@ private static void transformJson(
...
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "17:30", "18:20", true,
+                dateParser, dateFormatterBlocks, blocks);
+
+        addBlock("Closing ceremony", "Sala Londra", day, "18:20", "18:40", false,
+                dateParser, dateFormatterBlocks, blocks);
+    }
+
+    private static void addBlock(
+            @NonNull String title,
```
Comment (rock3r) [inline android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java:320]: Ah dude you even put the effort of annotating the params! Nothing else in the codebase does lol

Comment (Takhion) [inline android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java:320]: Hehe yeah it's a forced habit :grinning:

Comment (tiwiz) [inline android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java:320]: _cough_ brackets _cough_

**Pull request merged**